_posts/2017-08-10-glpi-atoum.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "GLPI migrate to atoum"
+date: 2017-08-10 10:00:00
+author: "@Grummfy"
+categories: news
+---
+
+The [GLPI project][GLPI] [recently announced][announced-en][^1] that they have migrated to atoum for their unit test suite. So let's get back on
+this news. We ask several questions to the official team of the project to have some feedback, [Johan Cwiklinski](https://github.com/trasher) 
```

Comment:
We interviewed Johan Cwiklinski and some other members of the project official team about this migration.